release-notes/supported-os.json

```diff
@@ -0,0 +1,121 @@
+{
+    "$schema": "https://json.schemastore.org/dotnet-supported-os.json",
+    "operating-systems": [
+        {
+            "name": "Ubuntu",
+            "lifecycle-policy": "https://wiki.ubuntu.com/Releases",
+            "last-updated": "2023-02-28",
+            "upcoming-eos": {
+                "cycle": "23.10",
+                "eos-date": "2024-07-01"
+            },
+            "cycles": [
+                {
+                    "cycle": "16.04",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2021-04-02"
+                },
+                {
+                    "cycle": "18.04",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0", "8.0"],
+                    "eos-date": "2023-05-31"
+                },
+                {
+                    "cycle": "20.04",
+                    "supported-channels": ["6.0", "7.0", "8.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2022-10-01"
+                },
+                {
+                    "cycle": "22.04",
+                    "supported-channels": ["6.0", "7.0", "8.0", "9.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2024-09-30"
+                },
+                {
+                    "cycle": "22.10",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2023-07-20"
+                },
+                {
+                    "cycle": "23.04",
+                    "supported-channels": [],
+                    "unsupported-channels": ["6.0", "7.0"],
+                    "eos-date": "2024-01-20"
+                },
+                {
+                    "cycle": "23.10",
+                    "supported-channels": ["6.0", "7.0", "8.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2024-07-01"
+                },
+                {
+                    "cycle": "24.04",
+                    "supported-channels": ["6.0", "7.0", "8.0", "9.0"],
+                    "unsupported-channels": [],
+                    "eos-date": "2029-07-01"
+                }
+            ]
+        },
+        {
+            "name": "iOS",
+            "lifecycle-policy": "https://support.apple.com/ios",
+            "last-updated": "2023-02-28",
+            "upcoming-eos": {
+                "cycle": "15",
+                "expected-eos-date": "2024-10-01"
```

Comment:
Added `expected-eos-date` in difference to `eos-date` as Apple is not announcing them ahead of time. So the question is, do we want to put the expected date as it might help with setting triggers when to recheck, or should we just leave it away completely?